shell/platform/fuchsia/flutter/tests/integration/touch-input/embedding-flutter-view/lib/embedding-flutter-view.dart

```diff
@@ -10,7 +10,6 @@ import 'dart:ui';
 import 'package:args/args.dart';
 import 'package:fidl_fuchsia_ui_app/fidl_async.dart';
```

Comment:
How are you going to handle "fuchsia.ui.app" and "fuchsia.ui.views"?

I think we could do:
* "_launchGfxChildView" connects to the incoming ViewProvider service, so it can be made into a platform message.
* The ViewHolderToken and ViewRef/ViewRefControl types can all be encoded as integers in the platform message (cast the zx_handle_t to an integer).  This is already what we do for the "View.create" message on the platform views channel so there is precedent. When the associated dart EventPair/etc objects die, they will call zx_handle_close but then ignore the error that gets returned.

Reply:
I'm going to update this PR with those changes too. Basically I expose a native function over platform messages that mints a child view (either gfx or flatland based on an argument) and returns the view id as an integer.